core/platform/image/gae_image_services.py

```diff
@@ -67,7 +65,8 @@ def compress_image(image_content, scaling_factor):
         str. Returns the content of the compressed image.
     """
     if not constants.DEV_MODE:
-        height, width = get_image_dimensions(image_content)
+        img = Image.open(io.BytesIO(image_content))
+        width, height = img.height, img.width
```

Comment:
Do some manual checks with sample images of different formats to confirm that the width/height in the new implementation are the same as that in the old implementation.

(You can probably do this locally quite easily by doing both computations and asserting that the results are the same.)

Please post, in this comment, a report/screenshot of your results and the different tests you used to verify.

Reply:
The tests are added in this document starting at "Tests Run", all the information is also uploaded there including screenshots.

Reply:
https://docs.google.com/document/d/1VB8YVAE_JNNHwQ8p5H2Zo7mHtd7rn-B6J6DaG-TrVTw/edit